styles.css

```diff
@@ -6,6 +6,7 @@ body {
 
 header, footer, aside, .main {
   padding: 1em;
+  margin: 0;
```

Comment:
Your selector should be specifically targeting the direct children of `header, footer, .main/main, aside`. That means you need to use `>`, which indicates a direct relationship between parent and child...

```css
header, footer, aside, .main > * {
  margin: 0;
}
```
> The `*` indicates we are targeting all direct children, regardless of type/class/id/etc